src/analysis/retail/monk/mistweaver/normalizers/EventLinks/HeroTalentEventLinks.ts

```diff
@@ -0,0 +1,21 @@
+import SPELLS from 'common/SPELLS';
+import { TALENTS_MONK } from 'common/TALENTS';
+import { EventLink } from 'parser/core/EventLinkNormalizer';
+import { EventType } from 'parser/core/Events';
+import { CAST_BUFFER_MS, STRENGTH_OF_THE_BLACK_OX } from './EventLinkConstants';
+
+export const HERO_TALENT_EVENT_LINKS: EventLink[] = [
+  {
+    linkRelation: STRENGTH_OF_THE_BLACK_OX,
+    linkingEventId: SPELLS.STRENGTH_OF_THE_BLACK_OX_BUFF.id,
+    linkingEventType: EventType.RemoveBuff,
+    referencedEventId: SPELLS.STRENGTH_OF_THE_BLACK_OX_SHIELD.id,
```

Comment:
Maybe have this link to env cast completion - on the very off chance that a buff expires the same time that the unity sotbo shield applies it would be a false positive